projects/packages/sync/src/class-main.php

```diff
@@ -24,6 +24,9 @@ public static function configure() {
 		if ( Actions::sync_allowed() ) {
 			add_action( 'plugins_loaded', array( __CLASS__, 'on_plugins_loaded_early' ), 5 );
 			add_action( 'plugins_loaded', array( __CLASS__, 'on_plugins_loaded_late' ), 90 );
+			// Init after plugins loaded and very early in the `init` action. This helps with issues where plugins init
+			// with a high priority or sites that use alternate cron.
+			add_action( 'init', array( 'Automattic\Jetpack\Sync\Actions', 'init' ), 1 );
```

Comment:
I'm really curious how Sync sending still works with the PR applied because in `Automattic\Jetpack\Sync\Actions::init` we introduce other `init` hooks [here](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/sync/src/class-actions.php#L114) and [here](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/sync/src/class-actions.php#L149) 🤔

Reply:
So these would work because the priority within those hooks is lower. However I have also tested changing the above to `add_action( 'after_setup_theme', array( 'Automattic\Jetpack\Sync\Actions', 'init' ) );`, so it still runs before `init` but after translations are loaded, and that still prevents the translation error triggering, and still allows changes to be synced from my testing. I'd defer to you with sync so if you're more comfortable with that I could change it.

Reply:
Ah, just saw your separate comment.